modules/oxide/src/less/theme/globals/global-variables.less

```diff
@@ -32,6 +32,10 @@
 @panel-border-radius: 6px; // Dialogs, menues etc.
 @control-border-radius: 6px; // Buttons, input fields etc.
 
+// Focus outline
+@keyboard-focus-outline: 0 0 0 2px @color-tint; // Uses box shadow property
```

Comment:
Maybe it would be a better idea to only set `@keyboard-focus-outline-width` and `@keyboard-focus-outline-color` as global variables and leave the rest of the shadow properties out of configurable values? 

That would reduce the repetition, simplify the dark skin (a bit) and would remove the need for a separate `inset` variable — inset could be applied in the mixin itself if we pass a bool variable to it.

Reply:
I made the requested changes!